WooCommerce/Classes/ViewRelated/Orders/Order Creation/FlowCoordinator/NewSimplePaymentsNoticeViewModel.swift

```diff
@@ -0,0 +1,54 @@
+import SwiftUI
+
+final class NewSimplePaymentsNoticeViewModel {
+
+    private let simplePaymentsNoticeView: BottomSheetNoticeView
+
+    /// Redirects to `HubMenu`tabBar
+    ///
+    private var completionHandler: (() -> Void)? = {
+        guard let mainTabBarController = AppDelegate.shared.tabBarController else {
+            return
+        }
+        mainTabBarController.navigateTo(.hubMenu)
+    }
+
+    init() {
+        simplePaymentsNoticeView = BottomSheetNoticeView(
```

Comment:
This view logic should also go to the view controller in my opinion. From there it can access the localization strings, that can stay here in the view model :)

Reply:
Gotcha! I moved the init into the ViewController here: 9c9b1222af97cb6bb0b278996c1396e0a17b3953 , so now the ViewModel only has the button tapped logic, the icon data, and the localisation strings 👍